### PR TITLE
tweak layout for the story page

### PR DESCRIPTION
**Pull request by yochannah**
### Related issue

n/a

### Proposed change

layout tweakery

### Reviewers

- [ ] Reviewers selected

I think this isn't quite right yet but on track?

### CODE REVIEW

**Comment by yochannah** [comment]
layout showing story image to the right and basic info to the fleft, followed by the stroy

**Comment by yochannah** [comment]
this is what happens if wip is true: 
shows story with an notification that it is a work in progress

**Comment by Ismael-KG** [comment]
@yochannah, can you make it so WIPs just show a title, a photo, a year, and a message like:

> **This story is still a work in progress.** If you would like to help write it or know some useful resources that should be referenced, please share some thoughts in the following GitHub issue! [LINK]

File: _layouts/story.html [hunk 1]
```diff
@@ -3,44 +3,52 @@
 ---
 <div class="post">
 
-  <header class="post-header">
-    <h1 class="post-title">{{ page.title }}</h1>
-  </header>
+  {% if page.wip %}
+  <div class="notice wip">
+    This is a work in progress and may need additional edits to meet the protocols.
```
Comment (Ismael-KG) [inline _layouts/story.html:8]: **This story is still a work in progress.** If you would like to help write it or know some useful resources that should be referenced, please share some thoughts in the following GitHub issue!

**Review by Ismael-KG — CHANGES_REQUESTED** [review]

**Review by Ismael-KG — APPROVED** [review]
I am not sure what I am approving.

**Comment by yochannah** [comment]
okay this is wip conditions now: 
shows the wording suggested by ismael

**Comment by yochannah** [comment]
I added a title because otherwise what are we even updating? 
image showing th title added to the WIP text

**Pull request merged**